code/game/machinery/constructable_frame.dm

```diff
@@ -685,6 +685,33 @@ obj/item/weapon/circuitboard/rdserver
 							"/obj/item/weapon/stock_parts/scanning_module" = 1,
 							"/obj/item/weapon/stock_parts/console_screen" = 2)
 
+
+
+/obj/item/weapon/circuitboard/smartfridge/attackby(var/obj/item/weapon/G, var/mob/user)
+	if(issolder(G))
+		var/obj/item/weapon/solder/S = G
+		var/list/static/smartfridge_choices = list("Food smartfridge" = /obj/item/weapon/circuitboard/smartfridge/,
+											"Secure chemistry smartfridge" = /obj/item/weapon/circuitboard/smartfridge/medbay,
```

Comment:
Ah yes the horrible tab inconsistency.

See, the tab character doesn't have a well defined size. Most text editors put it at 4 characters wide. And most browsers put it at 8 characters wide.

So it's indented twice as much on GitHub.

How do you fix it? Simple!

Instead of indenting everything to fit up to the `list(` like this:

```dm
var/variablewithareallydamnlongname = list("a",
                                           "b"
)
```
You do this:
```dm
var/variablewithareallydamnlongname = list(
    "a",
    "b"
)
```

Instant readability!